app/frontend/wallet/constants.ts

```diff
@@ -30,6 +30,16 @@ export const NETWORKS: {[key: string]: Network} = {
     epochsToRewardDistribution: 4,
     minimalOutput: 1000000,
   },
+  MARY_TESTNET: {
+    name: 'mary-testnet',
+    networkId: NetworkId.TESTNET,
+    protocolMagic: ProtocolMagic.MARY_TESTNET,
+    eraStartSlot: 4492800, // 21600 slot x 208 epochs
+    eraStartDateTime: Date.parse('29 Jul 2020 21:44:51 UTC'),
+    ttlSlots: 3600,
```

Comment:
I think we can drop all ttl-related consts, since it's optional since Allegra, but I'll let u know

Reply:
Tbh, I think it's a nice feature to be able to know for sure that if anything happens with the tx, one hour later it's definitely not valid. I wouldn't remove this even tho its optional.

Reply:
arguably, we can extract that from the config though into a regular constant like defaultTtlSlots or something, I don't really see it being tweaked across network, getting just duplicated all over the different network configs